Delejez/Bosszúcska

```diff
@@ -0,0 +1,68 @@
+Italt mérgezni, csapba vérezni
+Lánccal folytani, bűzzel gyilkolni
+Anyád kurvája
+Csapba vérezni, italt mérgezni
+Lángok közt édes Klementin
+
+Dokit leszarni, csapba vérezni
+Áh ezaz *kacsint*, rajta túlleni
+Anyád kurvája
+Áh ezaz *kacsint*, dokit megölni, felégni
+
+Oh megtorlásom
+Része leszel a vetetésért
+Csak ki kell főzni, bébi
+Ah-ah-ah, ah-ah-ah-ah
+
```

Comment:
Ez kilóméterekkel jobb, mint a nem kikommentelt. Esetleg "készülőben van, bébi". A végén meg Á-á-á, á-á-á-á, azt elfelejtetted lefordítani.

Reply:
Csak azért nem rakom bele, mert akkor ki kell taláni, hogy legyen az utolsó versszakban az "Its in the taking, making, baking, faking, taking..."